data/md/legislature/Mike-Griffith-c3b6f47e-90e1-4abf-af49-c5c6fd7993a2.yml

```diff
@@ -0,0 +1,16 @@
+id: ocd-person/c3b6f47e-90e1-4abf-af49-c5c6fd7993a2
+name: Mike Griffith
+given_name: Mike
+family_name: Griffith
+email: mike.griffith@house.state.md.us
+image: http://mgaleg.maryland.gov/2023RS/images/griffith02.jpg
+party:
+- name: Republican
```

Comment:
@sroomf this person should already have a file, we just need to add:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 35A
They previously served in house district 35b from 01/07/2020-01/11/2023